src/node/desktop/test/integration/startup.int.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * startup.int.ts
+ *
+ * Copyright (C) 2021 by RStudio, PBC
+ *
+ * Unless you have received this program directly from RStudio pursuant
+ * to the terms of a commercial license agreement with RStudio, then
+ * this program is licensed to you under the terms of version 3 of the
+ * GNU Affero General Public License. This program is distributed WITHOUT
+ * ANY EXPRESS OR IMPLIED WARRANTY, INCLUDING THOSE OF NON-INFRINGEMENT,
+ * MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE. Please refer to the
+ * AGPL (http://www.gnu.org/licenses/agpl-3.0.txt) for more details.
+ *
+ */
+
+import { describe } from 'mocha';
+import { assert } from 'chai';
+import path from 'path';
+import { Application } from 'spectron';
+
+// Find path to electron binary in node_modules
+function getElectron(): string {
+  const base = './node_modules/electron/dist/';
+  switch (process.platform) {
+  case 'darwin':
+    return path.join(base, 'Electron.app/Contents/MacOS/Electron');
+  case 'linux':
+    return path.join(base, 'electron');
+  case 'win32':
+    return path.join(base, 'electron.exe');
+  default:
+    throw Error(`Unsupported platform: ${ process.platform }`);
+  }
+}
+
+// Find path to RStudio entrypoint
+function getMain(): string {
+  return path.join(__dirname, '../../dist/main/app.js');
+}
+
+describe('Startup and Exit', function () {
+  this.timeout(10000);
+
+  beforeEach(function () {
```

Comment:
Does this imply that a new Electron instance is created and torn down for every test? (If so I'm guessing we'll want to tweak this in the future?)

Reply:
Yes, we can decide how we want to chunk this up (e.g. could have all tests in a single test source file run against a single instance, etc).

Just getting something in there to build on. :-)